testing/frr/docker-start

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Not sure if this fiel belongs in the aport.

Reply:
OK, I'll take it out...  Thanks for all your help here, I'll fix, test and re-push today...